openpgp/packet/signature_test.go

```diff
@@ -76,4 +76,57 @@ func TestSignUserId(t *testing.T) {
 	}
 }
 
+func TestSignatureWithLifetime(t *testing.T) {
+	lifeTime := uint32(3600 * 24 * 30) // 30 days
+	sig := &Signature{
+		SigType:         SigTypeGenericCert,
+		PubKeyAlgo:      PubKeyAlgoRSA,
+		Hash:            0, // invalid hash function
```

Comment:
In the test this code was copied from, this served a purpose, but here it doesn't, since you're setting `Hash` to a valid hash function below before signing - so could you just set this to SHA256 immediately here?

Reply:
Thanks for the review, good catch. I removed the redundant Hash setting.

Reply:
:+1: Thanks!